frontend/src/components/common/SchedulingProgressBar.tsx

```diff
@@ -0,0 +1,28 @@
+import { Flex } from "@chakra-ui/layout";
+import { Step, Steps } from "chakra-ui-steps";
+import React from "react";
+
+import { SchedulingProgessBarProps } from "../pages/Scheduling/types";
+
+const steps = [{ label: "1" }, { label: "2" }, { label: "3" }, { label: "4" }];
+
+const SchedulingProgressBar = ({
+  activeStep,
+}: SchedulingProgessBarProps): JSX.Element => {
+  return (
+    <Flex flexDir="column" width="100%">
+      <Steps
+        activeStep={activeStep}
+        responsive={false}
+        colorScheme="teal"
+        size="sm"
+      >
+        {steps.map(({ label }) => (
+          <Step key={label} colorScheme="blue" />
```

Comment:
```suggestion
          <Step key={label} colorScheme="evergreen.100" isCompletedStep={label===activeStep.label} />
```
and pass in `activeStep` as an object or something
```typescript
type StepType = {
  label: string;
  stepNumber: number;
}
interface SchedulingProgressBarProps {
  activeStep: StepType;
}

```
and then instead pass the active step prop like this: 

```typescript 
...
<Steps
activeStep={activeStep.stepNumber}

```
The figma mocks don't have the previous steps as filled in, so we should pass a prop to not have the past ones be active by default.